src/fiatconnect/TransferStatusScreen.tsx

```diff
@@ -133,12 +133,31 @@ function SuccessOrProcessingSection({
     | FiatExchangeEvents.cico_fc_transfer_success_view_tx
     | FiatExchangeEvents.cico_fc_transfer_processing_view_tx
 
+  const getSettlementTimeString = (settlementEstimation: SettlementEstimation) => {
+    switch (settlementEstimation.settlementTime) {
+      case SettlementTime.LESS_THAN_ONE_HOUR:
+        return t(DESCRIPTION_STRINGS[SettlementTime.LESS_THAN_ONE_HOUR])
+      case SettlementTime.LESS_THAN_X_HOURS:
+      case SettlementTime.LESS_THAN_X_DAYS:
+        return t(DESCRIPTION_STRINGS[settlementEstimation.settlementTime], {
+          upperBound: settlementEstimation.upperBound,
+        })
+      case SettlementTime.X_TO_Y_HOURS:
+      case SettlementTime.X_TO_Y_DAYS:
+        return t(DESCRIPTION_STRINGS[settlementEstimation.settlementTime], {
+          lowerBound: settlementEstimation.lowerBound,
+          upperBound: settlementEstimation.upperBound,
+        })
+      default:
+        // Should not be reachable
+        return t('fiatConnectStatusScreen.success.baseDescription')
+    }
```

Comment:
```suggestion
  const getSettlementTimeString = ({ settlementTime, lowerBound, upperBound }: SettlementEstimation) => {
    return t(DESCRIPTION_STRINGS[settlementTime], { lowerBound, upperBound })
```
idk if this even compiles but if it does it would be nice. the switch seems more verbose than we might need

Reply:
It doesn't compile unfortunately, I agree that it's a bit verbose as-is